File: 203-remove-linked-list-elements/remove-linked-list-elements.java
/**
 * Definition for singly-linked list.
 * public class ListNode {
 *     int val;
 *     ListNode next;
 *     ListNode() {}
 *     ListNode(int val) { this.val = val; }
 *     ListNode(int val, ListNode next) { this.val = val; this.next = next; }
 * }
 */
class Solution {
    public ListNode removeElements(ListNode head, int val) {
        ListNode temp=head;
        List<Integer> list=new ArrayList<>();
        while(temp!=null){
            if(temp.val!=val)
            list.add(temp.val);
            temp=temp.next;
        }
        ListNode dummy=new ListNode(0);
        ListNode t=dummy;
        for(int l:list){
            t.next=new ListNode(l);
            t=t.next;
        }
        return dummy.next;
    }
}

File: 203-remove-linked-list-elements/remove-linked-list-elements.py
# Definition for singly-linked list.
# class ListNode(object):
#     def __init__(self, val=0, next=None):
#         self.val = val
#         self.next = next
class Solution(object):
    def removeElements(self, head, val):
        """
        :type head: Optional[ListNode]
        :type val: int
        :rtype: Optional[ListNode]
        """
        temp=head
        lst=[]
        while temp:
            lst.append(temp.val) if temp.val!=val else 0
            temp=temp.next
        
        dummy=ListNode(0)
        t=dummy
        for x in lst:
            t.next=ListNode(x)
            t=t.next
        return dummy.next